File: app/src/main/java/org/butterbrot/heve/ubill/BillApplication.java
package org.butterbrot.heve.ubill;

import android.app.Application;
import org.butterbrot.heve.ubill.entity.MyObjectBox;

import io.objectbox.BoxStore;


public class BillApplication extends Application {

    private BoxStore boxStore;

    @Override
    public void onCreate() {
        super.onCreate();
        boxStore = MyObjectBox.builder().androidContext(this).build();

    }

    public BoxStore getBoxStore() {
        return boxStore;
    }
}

File: app/src/main/java/org/butterbrot/heve/ubill/BillApplication.kt
package org.butterbrot.heve.ubill;

import android.app.Application
import io.objectbox.BoxStore
import org.butterbrot.heve.ubill.entity.MyObjectBox


class BillApplication: Application() {

    lateinit var boxStore: BoxStore

    override fun onCreate() {
        super.onCreate()
        boxStore = MyObjectBox.builder().androidContext(this).build()
    }

    override fun onTerminate() {
        super.onTerminate()
        boxStore.closeThreadResources()
    }
}